templates/zerver/help/configure-desktop-notifications.md

```diff
@@ -29,3 +29,66 @@ the future.
 
 2. Under **Private messages and @-mentions**, select **Visual desktop notifications**
 and/or **Audible desktop notifications**.
+
+## Troubleshooting
+
+You may need to adjust your browser or system notification settings in order
+to allow Zulip to send you desktop notifications.
+
+### Zulip Desktop App on Mac OS
+
+1. From your desktop, open **System Notifications**.
```

Comment:
@rishig I think this should be **System Preferences**.

Reply:
Addressed in https://github.com/zulip/zulip/pull/9865